src/RapidezStatamicServiceProvider.php

```diff
@@ -249,6 +251,17 @@ public function bootSitemaps(): static
         return $this;
     }
 
+    public function bootStaticCaching(): static
+    {
+        if (config('statamic.static_caching.invalidation.class')) {
+            return $this;
+        }
+        
+        config()->set('statamic.static_caching.invalidation.class', CustomInvalidator::class);
+
+        return $this;
+    }
```

Comment:
Maybe flip it around so there is one return:

```suggestion
    public function bootStaticCaching(): static
    {
        if (!config('statamic.static_caching.invalidation.class')) {
            config()->set(
                'statamic.static_caching.invalidation.class',
                CustomInvalidator::class
            );
        }
        
        return $this;
    }
```